tutorials/plot_ecog.py

```diff
@@ -45,3 +48,23 @@
 subjects_dir = mne.datasets.sample.data_path() + '/subjects'
 fig = plot_trans(info, trans=None, subject='sample', subjects_dir=subjects_dir)
 mlab.view(200, 70)
+
+###############################################################################
+# Sometimes it is useful to make a scatterplot for the current figure view.
+# This is best accomplished with matplotlib. We can capture an image of the
+# current mayavi view, along with the xy position of each electrode, with the
+# `snapshot_brain_montage` function.
+
+xy, im = snapshot_brain_montage(fig, mon)
+# Convert from a dictionary to array to plot
+xy_pts = np.vstack(xy[ch['ch_name']] for ch in info['chs'])
```

Comment:
if you are using `ch_name` here then maybe it's better to `xy[ch] for ch in info['ch_names']` ?